packages/contracts-bedrock/contracts/libraries/Hashing.sol

```diff
@@ -71,9 +71,9 @@ library Hashing {
             return hashCrossDomainMessageV0(_target, _sender, _data, _nonce);
         } else if (version == 1) {
             return hashCrossDomainMessageV1(_nonce, _sender, _target, _value, _gasLimit, _data);
-        } else {
-            revert("Hashing: unknown cross domain message version");
```

Comment:
I don't think this is unreachable. A user might just provide a bogus message that triggers this.

Reply:
Yeah, I was thinking this was still used by one of the production contracts which would constrain the input, but the only usage of this high-level function is in the tests. Closing.

Reply:
Yeah, I was thinking this was still used by one of the production contracts which would constrain the input, but the only usage of this high-level function is in the tests. Closing.